package.json

```diff
@@ -35,5 +42,5 @@
     "moment": "^2.24.0",
     "popper.js": "^1.15.0",
     "vue": "^2.6.10"
-  }
+  },
```

Comment:
Are you sure this works?

Reply:
I mean this is package-json have to be in JSON format and `,` in the end is a mistake